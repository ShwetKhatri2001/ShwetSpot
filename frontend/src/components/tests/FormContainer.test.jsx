import React from 'react';
import renderer from 'react-test-renderer';

import FormContainer from '../FormContainer';

it('should render correctly', () => {
  const tree = renderer.create(<FormContainer />).toJSON();
  expect(tree).toMatchSnapshot();
});
