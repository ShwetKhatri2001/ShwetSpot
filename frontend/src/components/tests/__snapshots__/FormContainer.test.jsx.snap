// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="container"
>
  <div
    className="justify-content-md-center row"
  >
    <div
      className="col-md-6 col-12"
    />
  </div>
</div>
`;
